bootstrap.py

```diff
@@ -35,7 +35,7 @@
 Simply run this script in a directory containing a buildout.cfg, using the
 Python that you want bin/buildout to use.
 
-Note that by using --find-links to point to local resources, you can keep 
+Note that by using --find-links to point to local resources, you can keep
```

Comment:
can you revert all changes to the file?
it's a boilerplate file that shouldn't be modified at all (despite the trailing spaces i dislike as well :))